GitCommands/Git/GitModule.cs

```diff
@@ -790,15 +790,15 @@ public async Task<List<ConflictData>> GetConflictsAsync(string filename = "")
             return _gitTreeParser.ParseSingle(output);
         }
 
-        public int? GetCommitCount(string parentHash, string childHash)
+        public int? GetCommitCount(string parent, string child, bool cache = false)
         {
             var args = new GitArgumentBuilder("rev-list")
             {
-                parentHash,
-                $"^{childHash}",
+                parent,
+                $"^{child}",
                 "--count"
             };
-            var output = _gitExecutable.GetOutput(args);
+            var output = _gitExecutable.GetOutput(args, cache: cache ? GitCommandCache : null);
```

Comment:
How about this option?
To separate by parameters.

```
var output = cache switch
{
    true => _gitExecutable.GetOutput(args, cache: GitCommandCache),
    false => _gitExecutable.GetOutput(args)
};
```

Reply:
For this PR, I suggest to follow the current code style (i.e. not change).

But this could be changed in general.
I prefer the current ?: syntax, of course partly because I am used to that but I see no benefit with the switch here.
The current is easier to read for me and less duplicated code. Long commands where options wrap over several lines can be hard to compare.
This scales better when there are more options affecting a command too.

Reply:
> This scales better when there are more options affecting a command too.

Could you please to show example there is you mean.

Reply:
```c#
            return _gitExecutable.GetOutput(
                new GitArgumentBuilder("diff")
                {
                    "--no-color",
                    "-M -C",
                    "--name-status",
                    { nullSeparated, "-z" },
                    _revisionDiffProvider.Get(firstRevision, secondRevision)
                },
                cache: noCache ? null : GitCommandCache);
```

Reply:
In the case of the switch, the state machine is clearer.
And the appearance of "null" can signal a potential error.